docs/lake/design.md

```diff
@@ -7,7 +37,7 @@ with the [Zed data model](../formats).
 
 ## Data Pools
 
-A lake is comprised of _data pools_.  Each data pool is organized
+A lake is composed of _data pools_.  Each data pool is organized
```

Comment:
"Comprised of" appeared many times through the doc, and was flagged by the grammar checker, so I've changed it to the convention I learned about via [https://www.grammarly.com/blog/comprise-vs-compose/.](https://www.grammarly.com/blog/comprise-vs-compose/.) :man\_shrugging:  :smile:

Reply:
Or get fancy: `A like comprises _data pools_.`